src/lib/mongoBackend/dbConstants.h

```diff
@@ -80,6 +80,7 @@
 #define ENT_LOCATION_ATTRNAME        "attrName"
 #define ENT_LOCATION_COORDS          "coords"
 #define ENT_LAST_CORRELATOR          "lastCorrelator"
+#define ENT_EXPIRATION               "dateExpiration"
```

Comment:
To aling with other similar fields in databe, maybe this could be `expDate`.

Note this will not be the name used at API level (e.g. GET /v2/entities/foo?attrs=dateExpires`) which should use an style more aligned with NGSIv2 (e.g. `dateCreated` and `dateModified`).

Reply:
Fixed in 5a4703d